pkgs/misc/opcua-client-gui/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, python3
+, fetchFromGitHub
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "opcua-client-gui";
+  version = "0.8.4";
+
+  src = fetchFromGitHub {
+    owner = "FreeOpcUa";
+    repo = pname;
+    rev = version;
+    hash = "sha256-0BH1Txr3z4a7iFcsfnovmBUreXMvIX2zpZa8QivQVx8=";
+  };
+
```

Comment:
Missed:
```nix
nativeBuildInputs = [ copyDesktopItems ];
```

Reply:
Should I open a new PR adding it?

Reply:
No, it's not so major issue, just not forget to fix with next update.

Reply:
Okay allready made the change to my fork so I'm quite sure I won't forget ^^